main.py

```diff
@@ -156,10 +186,10 @@ def GetScoreForWord(Word, TileDictionary):
 def UpdateAfterAllowedWord(Word, PlayerTiles, PlayerScore, PlayerTilesPlayed, TileDictionary, AllowedWords):
     PlayerTilesPlayed += len(Word)
     for Letter in Word:
-        if PlayerTiles.find(Letter)>=0:
-          PlayerTiles = PlayerTiles.replace(Letter, "", 1)
+        if PlayerTiles.find(Letter) >= 0:
+            PlayerTiles = PlayerTiles.replace(Letter, "", 1)
```

Comment:
Extra indent

Reply:
Pycharm refactor, looks like a good change though. This makes the style consistent.